modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -763,10 +763,22 @@ class CV_EXPORTS_W QRCodeEncoder {
 
 };
 
+enum QRDetectMethod{
+    SEARCH_LINES, // old method
+    ARUCO // new method
+    // could add WECHAT method
+};
+struct CV_EXPORTS_W_SIMPLE QRDetectorParameters {
+    QRDetectorParameters() {}
+
+    CV_PROP_RW QRDetectMethod qrDetectMethod = QRDetectMethod::ARUCO;
```

Comment:
Do you plan to add new parameters to the algorithm? If not, the struct is redundant. Simple constructor parameter is enough.

Reply:
added new params